src/pyspiffe/bundle/jwt_bundle/exceptions.py

```diff
@@ -19,7 +31,7 @@ def __init__(self, key_id: str = "not specified") -> None:
         Args:
             key_id: the key_id with no authority associated.
```

Comment:
nit: I know this is not your change but I found this and so... kei_id: The key_id ..... it should be capital T.